autorest/azure/environments.go

```diff
@@ -132,6 +164,14 @@ var (
 		ContainerRegistryDNSSuffix:   "azurecr.cn",
 		CosmosDBDNSSuffix:            "documents.azure.cn",
 		TokenAudience:                "https://management.chinacloudapi.cn/",
+		ResourceIdentifiers: ResourceIdentifier{
+			Graph:          "https://graph.chinacloudapi.cn/",
+			KeyVault:       "https://vault.azure.cn",
+			Datalake:       NotAvailable,
+			MicrosoftGraph: NotAvailable,
+			Batch:          "https://batch.chinacloudapi.cn/",
+			Storage:        NotAvailable,
```

Comment:
storage is not available in all clouds ?

Reply:
I suspect it is but couldn't find it anywhere.  Hoping @yangl900 or somebody has the answer.

Reply:
ok. Im worried than that saying `NotAvailable` might send the wrong message that storage is not available in that cloud.

Reply:
this indicates storage-aad support is not available in all clouds.

Reply:
Is it?  I can't find any documentation on what the resource ID should be.